### DISCUSSION TITLE
Mac M系列，微调是不是效率很弱

### DISCUSSION THREAD

**Discussion by 100ZZ** [THUDM/GLM-4#223] [General]
M2MAX，MPS加速相关已设置；数据集就塞了一条github的例子；贼慢
微调训练这些，Mac是不是没法愉快玩耍

**Comment by zRzRzRzRzRzRzR** [comment]
是的，很慢的，至少开源的这套方案是非常慢的，如果有兴趣可以去看看MLX，但这个框架没有适配GLM，得自己适配
通常来说微调还是要用英伟达的显卡
